packages/ai/self_heal.py

```
@@ -83,12 +83,41 @@ async def self_heal_brain_and_unblock_tasks() -> dict[str, Any]:
+            store = await get_brain_config_store()
+            patch = BrainConfigPatch(
+                primary_provider="nvidia",
+                planner_model="z-ai/glm-5.2",
+                executor_model="z-ai/glm-5.2",
+                verifier_model="z-ai/glm-5.2",
+                judge_model="z-ai/glm-5.2",
+            )
+            await store.set_brain_config(patch, actor="self_heal_stale_provider_reset")
+            summary["failover_persisted"] = "nvidia"
+            summary["reset_reason"] = f"provider {normalized_active} not in known-good set"
+        except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)